tests/azure/templates/galaxy_pytest_script.yml

```diff
@@ -0,0 +1,61 @@
+---
+parameters:
+  - name: build_number
+    type: string
+  - name: scenario
+    type: string
+    default: fedora-latest
+  - name: ansible_version
+    type: string
+    default: ""
+  - name: python_version
+    type: string
+    default: 3.x
+
+jobs:
+- job: Test_PyTests
+  displayName: Run pytests on ${{ parameters.scenario }}
+  timeoutInMinutes: 120
+  steps:
+  - task: UsePythonVersion@0
+    inputs:
+      versionSpec: '${{ parameters.python_version }}'
+
+  - script: |
+      pip install \
+        "molecule[docker]>=3" \
+        "ansible${{ parameters.ansible_version }}"
+    displayName: Install molecule and Ansible
+
+  - script: ansible-galaxy collection install community.docker ansible.posix
+    displayName: Install Ansible collections
+
+  - script: pip install -r requirements-tests.txt
+    displayName: Install dependencies
+  - script: |
+      utils/build-galaxy-release.sh
+      ansible-galaxy collection install freeipa-ansible_freeipa-$(git describe --tags 2>/dev/null | sed -e "s/^v//").tar.gz -vvv
+      molecule create -s ${{ parameters.scenario }}
+    displayName: Setup test container
+    env:
+      ANSIBLE_LIBRARY: ./molecule
+
+  - script: |
+      mkdir /tmp/ansible-freipa
+      tar xzvf freeipa-ansible_freeipa-$(git describe --tags 2>/dev/null | sed -e "s/^v//").tar.gz -C /tmp/ansible-freeipa
```

Comment:
Can't we simply use the collection here that has been installed in the previous strep?

Reply:
It might be possible. I changed the scripts to use the collection tests.